linajea/candidate_database.py

```diff
@@ -211,36 +207,34 @@ def set_parameters_id(self, parameters_id):
         self.selected_key = 'selected_' + str(self.parameters_id)
         logger.debug("Set selected_key to %s" % self.selected_key)
 
-    def get_score(self, parameters_id, frames=None):
+    def get_score(self, parameters_id, eval_params=None):
```

Comment:
@abred I think we need to finally get rid of this frames hack, and not make it worse by passing in a whole eval_params object that is used to also update the score. Any eval params used to update the score record should be read from the database, not passed in in a way that could make a mismatch between the actual parameters used to calculate the score and those stored with the score. I'm rerunning all my experiments as we speak, and will be happy to drop the frames encoded into the scores collection name if you are. If we do want backwards compatibility, I think we should keep passing in the frames explicitly.